### PR TITLE
feat: style fulfillment login loading screen

### PR DESCRIPTION
**Pull request by aleespryker**
Closes [HRZ-2814](https://spryker.atlassian.net/browse/HRZ-2814)

[HRZ-2814]: https://spryker.atlassian.net/browse/HRZ-2814?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.component.ts [hunk 1]
```diff
@@ -26,8 +29,23 @@ export class OauthHandlerComponent extends LitElement {
   }
 
   protected render(): unknown {
-    return html`<p><oryx-spinner></oryx-spinner></p>
-      <p style="text-align: center;">${i18n('oauth.logging-you-in')}</p>`;
+    return html`
+      <div class="logo">
```
Comment (Sinucid) [inline libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.component.ts]: Can we drop not necessary wrappers and keep simpler structure? Like:
```
<oryx-image resource="logo"></oryx-image>
<oryx-heading><h3>${i18n('oauth.logging-you-in')}</h3></oryx-heading>
<oryx-spinner size="xl"></oryx-spinner>
```

File: libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.component.ts [hunk 2]
```diff
@@ -26,8 +29,23 @@ export class OauthHandlerComponent extends LitElement {
...
+        <oryx-image
+          resource="logo"
+          style="color: var(--oryx-color-canvas-100);"
```
Comment (Sinucid) [inline libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.component.ts]: Move, please, colors to the css styles

File: libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    flex-direction: column;
+    background-color: var(--oryx-color-primary-300);
+    justify-content: center;
+    height: 100vh;
+    color: var(--oryx-color-canvas-100);
+    --oryx-icon-size: 32px;
+  }
+
+  :host:before {
```
Comment (Sinucid) [inline libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts:14]: What is the aim of this?

**Review by Sinucid — CHANGES_REQUESTED** [review]

File: libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts (hunk 3 above)
Comment (aleespryker) [reply libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts:14]: The handler component does not cover the entire page, and `html` and `body` tag css does not work inside the web component. So I had to use a pseudo element to color the rest of the page.

Comment (Sinucid) [reply libs/platform/auth/src/oauth/components/oauth-handler/oauth-handler.styles.ts:14]: Yeah, looks like a hack, but ok

**Review by Sinucid — APPROVED** [review]

**Pull request merged**